### DISCUSSION TITLE
Setting proxy to default page, instead of the 503

### DISCUSSION THREAD

**Discussion by syntaxgirl** [evertramos/nginx-proxy-automation#308] [General]
How to setup default page for proxy, so it doesn't lead to 503?

**Comment by evertramos** [comment]
Hi @syntaxgirl!

Nice reminder. I was missing that option in fresh install script. You now may add the option `-dh your_domain.com` or `--default-host=your_domain.com` in order to set up the default host as per nginx-proxy documentation:

[https://github.com/nginx-proxy/nginx-proxy#default-host](https://github.com/nginx-proxy/nginx-proxy#default-host)

Please remember that you must have the same domain configured as a VIRTUAL_HOST in a new container and running in the same network of the proxy in order to correctly redirect the request to this container.
